### DISCUSSION TITLE
getting issues ❗️❓ when using github action pr pull for attempting to upload bottle artifcats.

### DISCUSSION THREAD

**Discussion by ipatch** [Homebrew/discussions#2432] [Tap maintenance and Homebrew development] [answered]
i have PR labeled with `pr pull`, and have had little issues with it in the past (but have run into some edge cases). i built two bottles of opencascade for catalina and big sur that have the `debug` cmake flag enabled. i can see the path to to the `bottle.tar` file on my _self-hosted_ runner filesystem. ie.

```
/usr/local/Cellar/opencascade-debug@7.5.3-bottle.tar
```

1. the bottles built without any issues, [learn more][lnk1]

there does appear to be a warning at the bottom of that page in the _Annotations_ section of the page.

so the next step i've been doing to publish the bottles after the github action completes successfully is to label the PR with `pr-pull` as described in the homebrew documentation.

then the next gha is fired off `publish.yml` and i encounter [this error][lnk2]

Why am i encountering this error?

[lnk1]: <https://github.com/FreeCAD/homebrew-freecad/actions/runs/1442385577>
[lnk2]: <https://github.com/FreeCAD/homebrew-freecad/runs/4168698400?check_suite_focus=true#step:4:30>

**Comment by ipatch** [comment, marked as answer]
did a little more digging into this issue. and came across this,

https://github.com/FreeCAD/homebrew-freecad/runs/4160478895?check_suite_focus=true#step:13:40


i guess this _begs the question_, should homebrew _test-bot_ be setup in a way where if a formula build successfully and presents green check mark ✅ for succesfully building the formula but then manages to fail the `brew test` should the github action not report as failing? seems to be a bit of a false positive 🤨 (but i've been known to be wrong in the past) ☹️

**Reply by carlocab** [reply]
Nope, you're right. Test-bot used to do the right thing here, but I adjusted the behaviour a little. This was an unintended consequence of that.

**Reply by carlocab** [reply]
Should be fixed by https://github.com/Homebrew/homebrew-test-bot/pull/698.
